plugin/src/main/java/io/fabric8/maven/plugin/converter/DeploymentOpenShiftConverter.java

```diff
@@ -86,10 +86,18 @@ public HasMetadata convert(HasMetadata item) {
                 if (strategy != null) {
                     strategyType = strategy.getType();
                 }
-                if (openshiftDeployTimeoutSeconds != null && openshiftDeployTimeoutSeconds > 0 &&
-                        (Strings.isNullOrBlank(strategyType) || "Rolling".equals(strategyType))) {
-                    specBuilder.withNewStrategy().withType("Rolling").
-                        withNewRollingParams().withTimeoutSeconds(openshiftDeployTimeoutSeconds).endRollingParams().endStrategy();
+                if (openshiftDeployTimeoutSeconds != null && openshiftDeployTimeoutSeconds > 0) {
+                    if (Strings.isNullOrBlank(strategyType) || "Rolling".equals(strategyType)) {
+                        specBuilder.withNewStrategy().withType("Rolling").
+                            withNewRollingParams().withTimeoutSeconds(openshiftDeployTimeoutSeconds).endRollingParams().endStrategy();
+                    } else if (Strings.isNotBlank(strategyType)) {
```

Comment:
Couldn't this check be placed top so that the 'rolling' check would benefit for this, too ?